src/metrics/external-types.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {GoogleAuthOptions} from 'google-auth-library';
+
+export interface ExporterOptions {
+  /**
+   * Optional authentication options for Google services.
+   */
+  authOptions?: GoogleAuthOptions;
```

Comment:
Instead of this could we pass the "auth" object created in `index.ts` file. 

If we plan to modify to use "MetricServiceClient" then we could directly pass the clientOptions created in `index.ts` .

Reply:
Done.